packages/TesterApp/scripts/release.js

```diff
@@ -0,0 +1,35 @@
+const { execSync } = require('child_process');
+
+function runCommand(command) {
+  try {
+    execSync(command, { stdio: 'inherit' });
+  } catch (error) {
+    console.error(`Error executing command: ${command}`, error);
+    process.exit(1);
+  }
+}
+
+function buildIOS() {
+  runCommand('pnpm bundle:ios');
```

Comment:
let's clean the build output dir for ios here - only `build/generated` is cleaned everytime we build